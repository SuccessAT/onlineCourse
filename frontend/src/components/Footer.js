import React from 'react';
import CssBaseline from '@material-ui/core/CssBaseline';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';
import Container from '@material-ui/core/Container';
import Link from '@material-ui/core/Link';

function Copyright() {
  return (
    <Typography variant="body2" color="textSecondary" style={{ fontSize: 12}}>
      {'Copyright Â© '}
      <Link color="inherit" href="localhost:3000">
        Hogwarts Online
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
}

const useStyles = makeStyles((theme) => ({

  footer: {
    padding: theme.spacing(3, 9),
    marginTop: theme.spacing(3),
    backgroundColor:
      theme.palette.type === 'light' ? theme.palette.grey[200] : theme.palette.grey[800],
  },
}));

export default function StickyFooter() {
  const classes = useStyles();

  return (

      <footer className={classes.footer}>
        
          <Typography variant="body1" align="left" style={{ fontSize: 13, fontWeight: "bold"}}>Hogwarts Online</Typography>
          <Typography variant="body1" style={{ fontSize: 12}}>Magical Lane 4</Typography>
          <Typography variant="body1" style={{ fontSize: 12}}>12478 London</Typography>
          <Typography variant="body1" style={{ fontSize: 12}}>United Kingdom</Typography>
          <Copyright />
        
      </footer>
 
  );
}