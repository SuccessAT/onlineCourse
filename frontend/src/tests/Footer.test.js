import React from 'react';
import ReactDom from 'react-dom';
import  Footer  from '../components/Footer.js';
import renderer from 'react-test-renderer'


it("Footer rendering", ()=> {
    const div = document.createElement("div");
    ReactDom.render(<Footer />, div)
})

it("matches snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot()
})