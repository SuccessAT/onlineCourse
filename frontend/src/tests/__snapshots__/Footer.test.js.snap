// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<footer
  className="makeStyles-footer-1"
>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft"
    style={
      Object {
        "fontSize": 13,
        "fontWeight": "bold",
      }
    }
  >
    Hogwarts Online
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1"
    style={
      Object {
        "fontSize": 12,
      }
    }
  >
    Magical Lane 4
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1"
    style={
      Object {
        "fontSize": 12,
      }
    }
  >
    12478 London
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1"
    style={
      Object {
        "fontSize": 12,
      }
    }
  >
    United Kingdom
  </p>
  <p
    className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
    style={
      Object {
        "fontSize": 12,
      }
    }
  >
    Copyright Â© 
    <a
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
      href="localhost:3000"
      onBlur={[Function]}
      onFocus={[Function]}
    >
      Hogwarts Online
    </a>
     
    2020
    .
  </p>
</footer>
`;
